lib/msal-browser/src/interaction_handler/SilentHandler.ts

```diff
@@ -110,16 +110,13 @@ export class SilentHandler extends InteractionHandler {
          */
 
         return new Promise((resolve, reject) => {
-            const frameHandle = this.createHiddenIframe();
+            const frameHandle = this.createHiddenIframe(urlNavigate);
 
             setTimeout(() => {
                 if (!frameHandle) {
                     reject("Unable to load iframe");
                     return;
                 }
-
-                frameHandle.src = urlNavigate;
```

Comment:
This is a breaking change and needs to wait for v4 - can you file a work item if one doesn't exist?